docs/integrations/athena.md

```diff
@@ -12,12 +12,137 @@ redirect_from: ../using/athena.html
+1. Creates partitioned directories with symlink files in the underlying S3 bucket.
+1. Creates a table in Glue catalog with symlink format type and location pointing to the created symlinks.
 
-In order to support querying data from lakeFS with Amazon Athena, we will use [create-symlink](glue_hive_metastore.md#create-symlink), one of the [metastore commands](glue_hive_metastore.md) in [lakectl](../reference/commands.md).
+**Note**
+`.lakectl.yaml` file should be configured with the proper hive/glue credentials. [For more information](glue_hive_metastore.md#configurations) 
+{: .note }
 
-create-symlink receives a table in glue pointing to lakeFS and creates a copy of the table in glue pointing to the underlying S3 bucket.
-We can then query the new created table with Athena
+create-symlink receives a table in glue or hive pointing to lakeFS and creates a copy of the table in glue.
+The table data will use the `SymlinkTextInputFormat`, which will point to the lakeFS repository storage namespace. Without copying data,
+you will be able to query your data with Athena. However, the symlinks table will only show the data that existed during 
+the copy. If the table changed in lakeFS, you need to run `create-symlink` again for your changed to be reflected in Athena.
 
+### Example:
 
+Let's assume that some time ago, we created a hive table `my_table` that is stored in lakeFS repo `example` under branch `main`, using the command:
+```shell
+CREATE EXTERNAL TABLE `my_table`(
+   `id` bigint, 
+   `key` string 
+)
+PARTITIONED BY (YEAR INT, MONTH INT)
+LOCATION 's3://example/main/my_table';
+WITH (format = 'PARQUET', external_location 's3a://example/main/my_table' );
+```
+
+The repository `example` has the s3 storage space `s3://my-bucket/my-repo-prefix/`. 
+After inserting some data to it, object structure under `lakefs://example/main/my_table` looks like:
+
+![lakefs_table.png](../assets/img/lakefs_table.png)
+
+Now we would like to query that table with Athena. We need to use the `create-symlink` command as follows:
+
+```shell
+lakectl metastore create-symlink \
+--repo example \
+--branch main \
+--path my_table \
+--from-client-type hive \
+--from-schema default \
+--from-table my_table \
+--to-schema default \ 
+--to-table my_table  
+```
+
+The command will have 2 notable outputs:
+
+1. For each partition the command will create a symlink file:
+
+```shell
+➜   aws s3 ls s3://my-bucket/my-repo-prefix/my_table/ --recursive
+2021-11-23 17:46:29          0 my-repo-prefix/my_table/symlinks/example/main/my_table/year=2021/month=11/symlink.txt
+2021-11-23 17:46:29         60 my-repo-prefix/my_table/symlinks/example/main/my_table/year=2021/month=12/symlink.txt
+2021-11-23 17:46:30         60 my-repo-prefix/my_table/symlinks/example/main/my_table/year=2022/month=1/symlink.txt
+```
+
+An example content of a symlink file:
+```text
+s3://my-bucket/my-repo-prefix/b0165d5c5b13473d8a0f460eece9eb26
```

Comment:
I suppose adding some more rows should make it more clear to readers not familiar with symlink files.

Reply:
Done.